server/datastore/mysql/microsoft_mdm.go

```diff
@@ -137,7 +137,7 @@ func (ds *Datastore) MDMWindowsDeleteEnrolledDeviceWithDeviceID(ctx context.Cont
 
 // MDMWindowsDeletePendingCommands deletes pending commands for a given device id.
 func (ds *Datastore) MDMWindowsDeletePendingCommands(ctx context.Context, deviceID string) error {
-	stmt := "DELETE FROM windows_mdm_pending_commands WHERE device_id = ?"
+	stmt := "DELETE FROM old_windows_mdm_pending_commands WHERE device_id = ?"
```

Comment:
The code is still completely the same for the moment (it doesn't use the new schema), just the names of existing tables have been renamed to `old_...`.